index.html

```diff
@@ -12,60 +12,109 @@
+      <!----------------------->
+      <section id="booklist" class="container p-3">
         <h2 class="h4">List of all books</h2>
 
         <!-- Container of all books -->
-        <div class="container container--book">
-          <!--
-          <div class="row border border-success-subtle rounded-2 align-items-center p-2 fs-5 mt-2">
-            <div class="col col-12 col-md-8 mb-3 mb-md-0 text-center text-md-start">
-              <p class="book-title mb-0">
-                <strong>Book</strong> by <strong>Author</strong>
-              </p>
-            </div>
-            <div class="col col-12 col-md-4">
-              <button class="btn btn-danger w-100 fs-5">Remove</button>
-            </div>
+        <!-- The content of this section will be added by JavaScript -->
+        <div class="container container--book"></div>
+      </section>
+
+      <!----------------------->
+      <!-- SECTION  ADD BOOK -->
+      <!----------------------->
+      <section
+        id="addbook"
+        class="container p-5  align-items-end d-none"
+      >
+        <form class="row">
+          <h4 class="h4 text-center mb-3 col-12">Add a new book:</h4>
+          <div class="d-none d-md-block col-2"></div>
+          <div class="col d-flex flex-column gap-3 align-items-center">
+            <input
+              required
+              type="text"
+              id="book-title"
+              placeholder="Book Title"
+              class="form-control"
+            />
+
+            <input
+              type="text"
+              required
+              id="book-author"
+              placeholder="Book Author"
+              class="form-control"
+            />
+
+            <button id="add-book" class="btn btn-success align-self-end w-50">
+              Add
+            </button>
           </div>
-          -->
+          <div class="d-none d-md-block col-2"></div>
+        </form>
+      </section>
+
+      <!----------------------->
+      <!-- SECTION CONTACCT -->
+      <!----------------------->
+      <section id="contact" class="d-none">
+        <div class="container pt-3 pb-3 bg-secondary-subtle text-center">
+          <h2>Contact Information</h2>
+          <p>
+            Do you have any questions or you just want to say hello? you can
+            reach out to us!
+          </p>
+          <ul class="list-group">
+            <li class="list-group-item">
+              <strong>Our Eamil: </strong> anwarhussaini160@gmail.com
+            </li>
+            <li class="list-group-item">
+              <strong>Our Phone number: </strong> +93 76 459 7266
+            </li>
+            <li class="list-group-item">
+              <strong>Our Address: </strong> Nakamise-dori: 22, 8453 Tokyo,
+              Japane
+            </li>
+          </ul>
         </div>
       </section>
     </main>
+    <footer class="container bg-secondary-subtle p-2"><p class="container">All rights resereved.</p></footer>
```

Comment:
- [ ] **[OPTIONAL]** Kindly make the footer always stay at the bottom of the page even when there is no content or books on the page. Take a look at the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) and see that the footer is at the bottom of the page. Here is a [link](https://dev.to/nehalahmadkhan/how-to-make-footer-stick-to-bottom-of-web-page-3i14) to see how to make the footer to stay at the bottom of the page.